src/app/InteractionModelEngine_BACKUP_97967.cpp

```diff
@@ -0,0 +1,786 @@
+/*
```

Comment:
This file should not be here.  Similar for various other files in this dir, probably the `Commands.h` under `test/`, probably all the bits under `cluster/`, and likely the new `FibonacciUtils`.